repose-aggregator/tests/performance-tests/test_vars/filters/scripting.yml

```diff
@@ -1,3 +1,8 @@
+# TODO: Remove this file? Don't want it to get picked up by the automated performance tests, but
+# TODO: keeping this file gives us a sensibly named way to run generic scripting tests.
+#
+# TODO: YAML does not seem to support include or inherit. As such, there is more duplication than
+# TODO: necessary because each scripting language only changes the "script_lang" variable.
```

Comment:
Correct on all points. So I guess the use-case would be if you wanted to throw a lot of custom stuff at the performance test framework. For example, say you wanted to ad-hoc testing Jaskell with the scripting filter. You could pass a `script_lang` of `jaskell` and custom `config_files` and without following the conventions of the performance testing framework, you could run a test using Jaskell. In that case, it would be more confusing to have to do what you did in your last example -- customize the `script_lang` for a test already named with a scripting language. Of course, that's a pretty advanced example, and honestly, that test should probably just conform to the conventions instead.

Which is all to say, I agree with you, so I will delete this file.